addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
```

Comment:
That's an interesting point. I didn't change them as I wanted to minimize API changes.  Should I make them required?  I like @sandersky 's point about wanting an error or warning.